### PR TITLE
Update syncproxy.md

### PR DESCRIPTION
**Pull request by QuadrixYT**
Add the tip for Custom Tablist(Remove Cloud Tab)

### CODE REVIEW

File: versioned_docs/version-3.3/modules/syncproxy.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ This is a json array containing the different maintenance MOTDs which are random
 If you want to turn off maintenance MOTDs empty the array. This should look like this: `"maintenanceMotds": [],`
 :::
 
+#### tablist
+This is a json array containing the different tablists which are randomly selected.
```
Comment (0utplay) [inline versioned_docs/version-3.3/modules/syncproxy.md]: The tablist is not really random, it's an animation

Comment (QuadrixYT) [reply versioned_docs/version-3.3/modules/syncproxy.md]: Is fixed in new Commit

File: versioned_docs/version-3.3/modules/syncproxy.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ If you want to turn off maintenance MOTDs empty the array. This should look like
 :::
 
 #### tablist
-This is a json array containing the different tablists which are randomly selected.
+This is a json array containing the different tablists which are playing in a animation.
```
Comment (0utplay) [inline versioned_docs/version-3.3/modules/syncproxy.md]: an animation

File: versioned_docs/version-3.3/modules/syncproxy.md [hunk 3]
```diff
@@ -36,6 +36,12 @@ This is a json array containing the different maintenance MOTDs which are random
 If you want to turn off maintenance MOTDs empty the array. This should look like this: `"maintenanceMotds": [],`
 :::
 
+#### tablist
+This is a json array containing the different tablists which are playing in an animation.
```
Comment (derklaro) [inline versioned_docs/version-3.3/modules/syncproxy.md]: In my opinion

```suggestion
It is a json array of the different animations that are displayed in the tablist one after the other.
```

would fit better

**Review by derklaro — CHANGES_REQUESTED** [review]

File: versioned_docs/version-3.3/modules/syncproxy.md (hunk 3 above)
Comment (QuadrixYT) [reply versioned_docs/version-3.3/modules/syncproxy.md:40]: Ok i will change it to that

**Review by Sarsum — APPROVED** [review]
Looks fine, thank you!

**Pull request merged**